Misc/NEWS.d/next/Core and Builtins/2022-05-12-15-19-00.gh-issue-92734.d0wjDt.rst

```diff
@@ -0,0 +1 @@
+Add an indentation feature to ``reprlib.Repr``
```

Comment:
This NEWS entry is misplaced (it's not "Core", it's "Library") and it's somewhat terse. How about something like: 
```suggestion
Allow multi-element reprs emitted by :mod:`reprlib` to be pretty-printed using configurable indentation.
```

Reply:
My bad. Will fix the filename. Thanks for the improved phrasing. :+1:

Reply:
File has been moved f7e91b6cb5deec8bf637707b000c4f30e890c5bb